lib/interfaces/include/VCRCANInterfaceImpl.h

```diff
@@ -32,11 +32,29 @@ template <CAN_DEV_TABLE CAN_DEV> using FlexCAN_Type = FlexCAN_T4<CAN_DEV, RX_SIZ
 // this is being done to send immediately from the inverter CAN line to the TELEM CAN every inverter
 
 struct CANInterfaces {
-    explicit CANInterfaces(VCFInterface &vcf_int, DrivebrainInterface &db_int)
-        : vcf_interface(vcf_int), db_interface(db_int) {}
+    explicit CANInterfaces(
+        VCFInterface &vcf_int, 
+        DrivebrainInterface &db_int, 
+        InverterInterface &fl_inverter_int, 
+        InverterInterface &fr_inverter_int, 
+        InverterInterface &rl_inverter_int, 
+        InverterInterface &rr_inverter_int
+
```

Comment:
could this be a veh_vec of inverter interfaces? maybe I'll keep reading and realize it can't be, idk

Reply:
nah u right it can be, i changed it.